### DISCUSSION TITLE
Use quadprog-wheel to improve installation process?

### DISCUSSION THREAD

**Discussion by petsuter** [spectrochempy/spectrochempy#482] [Developer discussions]
Currently spectrochempy depends on quadprog. quadprog does not release wheels, so a C compiler is required to build the library.
This is error prone and often causes the installation of spectrochempy to fail.

[quadprog-wheel](https://pypi.org/project/quadprog-wheel/) provides wheels and apparently the preferred solution is to depend on quadprog-wheel instead of quadprog:
https://github.com/quadprog/quadprog/issues/15#issuecomment-1268277587

Could spectrochempy be changed to depend on quadprog-wheel?

Thanks.

**Comment by preeyan** [comment]
As of v0.1.11 quadprog now has prebuilt wheels, so this change may no longer be necessary.
